src/System.Transactions/src/System/Transactions/DependentTransaction.cs

```diff
@@ -41,7 +41,7 @@ public void Complete()
             {
                 if (Disposed)
                 {
-                    throw new ObjectDisposedException("Transaction");
+                    throw new ObjectDisposedException(nameof(Transaction));
```

Comment:
@jimcarley, is this also a bug in the original code, similar to your comment for `CommittableTransaction`? Should it be "DependentTransaction" here instead of "Transaction"?